EmailValidator/Validation/DNSCheckValidation.php

```diff
@@ -44,7 +44,13 @@ public function getWarnings()
 
     protected function checkDNS($host)
     {
-        $host = rtrim(idn_to_ascii($host, IDNA_DEFAULT, INTL_IDNA_VARIANT_UTS46), '.') . '.';
+        if ( defined('INTL_IDNA_VARIANT_UTS46') ) {
+            $variant = INTL_IDNA_VARIANT_UTS46;
+        } else {
```

Comment:
Instead of using `else` you can initialize `$variant` with the default value `INTL_IDNA_VARIANT_2003` and only leave the `if` block.

Reply:
I know, but `INTL_IDNA_VARIANT_2003` is deprecated, so it's an odd default value to have.

Reply:
The current code uses it as a default anyway. Also it's been recently deprecated (startingPHP 7.2) so might be more sensible default than `INTL_IDNA_VARIANT_UTS46`.

Reply:
@tomsommer  please change it to remove the `else` block so I can merge and create a release with all the other changes you have submitted.
Thanks!